From 43278894bd9f1277da14ac187ba7b42ece5cf483 Mon Sep 17 00:00:00 2001
From: Sandro <devel@penguinpee.nl>
Date: Sun, 8 Dec 2024 11:11:27 +0100
Subject: [PATCH 1001/1005] Revert "meson: Add check for Flatpak"

We still want to package Bottle as RPM for our users, despite upstream
having a very strong, almost hostile, preference for Flatpak.

This reverts commit 6206301992c3218060761e5b574b83da485b759a.
---
 bottles/frontend/meson.build | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/bottles/frontend/meson.build b/bottles/frontend/meson.build
index 6ff7c011..c26ea0b9 100644
--- a/bottles/frontend/meson.build
+++ b/bottles/frontend/meson.build
@@ -23,12 +23,6 @@ params_file = configure_file(
     configuration: conf
 )
 
-fs = import('fs')
-
-if not fs.is_file('/' + '.flatpak-info')
-  error('file does not exist')
-endif
-
 bottles_sources = [
   '__init__.py',
   'main.py',
-- 
2.51.0

